### ISSUE TITLE
Test  test_zerosize2  fails  with py7zr\py7zr.py:373: FileNotFoundError

### ISSUE THREAD

**Issue by miurahr** [miurahr/py7zr#24]
[CI test]9https://ci.appveyor.com/project/miurahr/py7zr/build/job/98iib8pdxy0brq8n) fails with a following error on [topic-zerofile branch ](https://github.com/miurahr/py7zr/tree/topic-zerofile-azure-pipeline);

```
_______________________________ test_zerosize2 ________________________________
    @pytest.mark.files
    def test_zerosize2():
        archive = py7zr.SevenZipFile(open(os.path.join(testdata_path, 'test_6.7z'), 'rb'))
        tmpdir = tempfile.mkdtemp()
>       archive.extractall(path=tmpdir)
tests\test_advanced.py:160: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
py7zr\py7zr.py:607: in extractall
    self._set_file_property(o, p)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
self = <py7zr.py7zr.SevenZipFile object at 0x000000BC8749AF98>
outfilename = 'C:\\Users\\appveyor\\AppData\\Local\\Temp\\1\\tmpi8fz4sea\\5.13.0/msvc2017_64/include/QtScript/qtscript-config.h'
properties = {'archivable': True, 'attributes': 1048608, 'compressed': 0, 'creationtime': ArchiveTimestamp(132052647074275978), ...}
    def _set_file_property(self, outfilename: str, properties: Dict[str, Any]) -> None:
        # creation time
        creationtime = ArchiveTimestamp(properties['lastwritetime']).totimestamp()
        if creationtime is not None:
>           os.utime(outfilename, times=(creationtime, creationtime))
E           FileNotFoundError: [WinError 2] The system cannot find the file specified: 'C:\\Users\\appveyor\\AppData\\Local\\Temp\\1\\tmpi8fz4sea\\5.13.0/msvc2017_64/include/QtScript/qtscript-config.h'
py7zr\py7zr.py:373: FileNotFoundError

```

**Comment by miurahr** [comment]
It is happened with [test data test_6.7z](https://github.com/miurahr/py7zr/blob/topic-zerofile-azure-pipeline/tests/data/test_6.7z) which is come from Qt package for windows.

**Comment by miurahr** [comment]
Fix merged #25
